### ISSUE TITLE
[Bug] Malware Clustering - py2neo

### ISSUE THREAD

**Issue by weslambert** [TheHive-Project/Cortex-Analyzers#983]
**Describe the bug**
py2neo fails to install for MalwareClustering analyzer.

**To Reproduce**
pip3 install -r CortexAnalyzers/analyzers/MalwareClustering/requirements.txt

**Expected behavior**
py2neo should install successfully.

**Complementary information**
```
ERROR: Could not find a version that satisfies the requirement py2neo==3.1.2 (from versions: 4.0.0, 4.1.0, 4.1.1, 4.1.2, 4.1.3, 4.2.0, 4.3.0, 2020.0.0, 2020.1.0, 2020.1.1, 2021.0.0, 2021.0.1)
ERROR: No matching distribution found for py2neo==3.1.2
```

https://github.com/TheHive-Project/Cortex-Analyzers/blob/master/analyzers/MalwareClustering/requirements.txt#L4

**Work environment**
Ubuntu. 18.04 in Docker
Malware Clustering 1.0

**Possible solutions**
Remove the version pin for py2neo in `requirements.txt`, changing to simply `py2neo`

**Comment by dadokkio** [comment]
Hello, I did an update for this analyzer in develop recently. 
Can you try if this version works better? https://github.com/TheHive-Project/Cortex-Analyzers/tree/develop/analyzers/MalwareClustering

**Comment by weslambert** [comment]
I haven't had a chance to test at the moment, as I wasn't actually using the analyzer.  It happened that the analyzer's requirements were causing the Docker build to fail, because the version of the package is no longer available.  I will try testing as I get time.

**Comment by nadouani** [comment]
@dadokkio If this has been included in `develop` branch, it will be released with 3.0.0.

Adding to the right milestone.
